tools/lkl/lib/virtio_net_dpdk.c

```diff
@@ -33,9 +33,14 @@ static char *ealargs[4] = {
 /* XXX: disable cache due to no thread-safe on mempool cache. */
 #define MEMPOOL_CACHE_SZ        0
 /* for TSO pkt */
-#define MAX_PACKET_SZ           (65535 \
+//For ixgbe and vmxnet3 drivers
+/*#define MAX_PACKET_SZ           (65535 \*/
+/*	- (sizeof(struct rte_mbuf) + RTE_PKTMBUF_HEADROOM))*/
+//For Intel i350 NIC (igb driver)
+#define MAX_PACKET_SZ           (16383 \
 	- (sizeof(struct rte_mbuf) + RTE_PKTMBUF_HEADROOM))
-#define MBUF_NUM                (512*2) /* vmxnet3 requires 1024 */
+//#define MBUF_NUM                (512*2) /* vmxnet3 requires 1024 */
+#define MBUF_NUM                4096 /* Intel i350 NIC (igb driver): 4096 */
```

Comment:
In this context, I would like to ask if the following solution is acceptable. One can create a macro something like

#TESTED_PLATFORM

and depending on its value to set the MBUF_NUM and MAX_PACKET_SZ.

Could you please tell if you prefer another solution? Maybe a config file could work better?